### PR TITLE
Update notification message for delete project

### PR DESCRIPTION
**Pull request by aeng**
https://zanata.atlassian.net/browse/ZNTA-1066

### CODE REVIEW

File: zanata-war/src/main/resources/messages.properties [hunk 1]
```diff
@@ -374,6 +374,7 @@ jsf.project.MaintainerRemoved=Maintainer "{0}" has been removed from project.
 jsf.project.NeedAtLeastOneMaintainer=Need at least 1 maintainer in project.
 jsf.project.MaintainerAdded=Maintainer "{0}" has been added to project.
 jsf.project.status.updated=Updated project status "{0}".
+jsf.project.status.deleted=Deleted
```
Comment (djansen-redhat) [inline zanata-war/src/main/resources/messages.properties]: Maybe "Project deleted"?

Comment (aeng) [inline zanata-war/src/main/resources/messages.properties]: Good point.

**Comment by huangp** [comment]
:+1:

**Comment by definite** [comment]
@zanata-jenkins retest this please

**Comment by definite** [comment]
:+1:

**Pull request merged**